datadict/datadict-postgres.inc.php

```diff
@@ -311,18 +311,27 @@ function _recreate_copy_table($tabname,$dropflds,$tableflds,$tableoptions='')
 	{
 		if ($dropflds && !is_array($dropflds)) $dropflds = explode(',',$dropflds);
 		$copyflds = array();
-		foreach($this->MetaColumns($tabname) as $fld) {
+		foreach(($meta=$this->MetaColumns($tabname)) as $fld) {
```

Comment:
What is the purpose of this assignment ? `$meta` does not appear to be used anywhere.